hbase-server/src/test/java/org/apache/hadoop/hbase/master/cleaner/TestLogsCleaner.java

```diff
@@ -314,4 +318,24 @@ public ZKWatcher getZooKeeper() {
       return null;
     }
   }
+
+  private static class FSTableDescriptorsTest extends FSTableDescriptors {
```

Comment:
Do we really need this class? Just use Mockito to mock the return value of exists method is enough?

Reply:
> Do we really need this class? Just use Mockito to mock the return value of exists method is enough?

Yes, it can be done.  And the new implementation is in the new commit.